CONTRIBUTING.md

```diff
@@ -0,0 +1,444 @@
+# Contributing to CG
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+This is a guide for contributing to the CG package. Please check here first if you want to [set up an environment and develop](#local-development), [open and issue](#reporting-bugs), [suggest an enhancement](#suggesting-enhancements), [open a pull request](#pull-requests) etc.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[Branch Model](#branch-model)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Python style guide](#python-styleguide)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+[CG Design decisions](#design-decisions)
+  * [apps](#apps)
+  * [CLI](#cli)
+  * [meta](#meta)
+  * [server](#sever)
+  * [store](#store)
+
+
+
+
+## Code of Conduct
+
+Communicating around code can be a sensitive thing so please do your best to keep a positive tone. Remember that people are putting significant amount of work behind a PR or a review, stay humble :star:.
```

Comment:
```suggestion
Communicating around code can be a sensitive thing so please do your best to keep a positive tone. Remember that people are putting significant amount of work behind a PR or a review, stay humble :star:
```
Remove the dot after icon when the icon ends the sentence and the sentence ends the section, ditto all occurrences

Reply:
Fixed that